src/types.ts

```diff
@@ -0,0 +1,36 @@
+import { Domain } from "@zero-tech/data-store-core";
+
+export interface DomainsApiResponse {
+  numResults?: number;
+  results: Domain[];
```

Comment:
Would `domains` be a better name for this?

Reply:
I think it would prefer to keep it generic, that way one day we get instead return a generic pagination response without changing the existing parts of the payload